index.markdown

```diff
@@ -424,7 +424,7 @@ How do we store information on a array?
 
 ### `setitem` (opcode `0xc4`)
 
-Opcode `setitem` consumes an Array pointer `p`, an index `i` and a value `v` from stack, and performs the following attribution: `(*p)[i] = v`.
+Opcode `setitem` consumes an Array pointer `p`, an index `i` and a value `v` from stack, and performs the following attribution: `p[i] = v`.
```

Comment:
In fact such a notation doesn't exist... I put "c-style" here, to demonstrate that pointer is being materialized on memory, before access (`operator*`). Do you think direct p[i] access is clearer?

Reply:
The way I understood that when I read is that first p is dereferenced with `*` and then with `[i]` so it must mean that p is a pointer to a pointer, which it seems is not the case.
But maybe you were trying to convey something else I missed or this is aimed at an audience which is not me.

Reply:
I agree with you: `p[i]` is correct.